### ISSUE TITLE
An updated release for React 16

### ISSUE THREAD

**Issue by spencewood** [NYTimes/react-tracking#76]
I really appreciate you putting this library out there!

I'm a little stuck at the moment though because, while the latest published version does indeed support React 16, the peer dependency of `react@^15.0.0` doesn't meet semver package requirements when using it alongside React 16, and a `yarn check` yields the following with `react-tracking@5.2.1`:

```
...
error "package#react-tracking#react@^15.0.0" doesn't satisfy found match of "react@16.4.0"
...
```

It looks to be all fixed up with with https://github.com/NYTimes/react-tracking/pull/72, so I was hoping a new release could be published that matches what's in master. Thanks!

**Comment by tizmagik** [comment]
Ah, yes sorry about that. Will get a release out shortly!

**Comment by tizmagik** [comment]
Released [v5.3.0](https://github.com/NYTimes/react-tracking/releases/tag/v5.3.0)
